lib/ignore/ignore.go

```diff
@@ -411,7 +411,7 @@ func loadParseIncludeFile(filesystem fs.Filesystem, file string, cd ChangeDetect
 
 	fd, info, err := loadIgnoreFile(filesystem, file)
 	if err != nil {
-		return nil, err
+		return nil, parseError(fmt.Errorf("could not load included ignore file: %v", err))
```

Comment:
Should use to use `%w` to preserve erro chain. And the inner error already signifies that something went wrong, the wrapping context could be more terse to shrink the overall length:

```suggestion
		return nil, parseError(fmt.Errorf("included ignore file: %w", err))
```

Reply:
Actually the error is already wrapped in the calling function, so I'd say no wrapping/context is needed here at all: https://github.com/syncthing/syncthing/blob/3d94efd42785b207ddca3fb9af75d54dd863e178/lib/ignore/ignore.go#L561

Reply:
So there should be a comment to this effect, but the entire point of this fix is to *not* preserve the error chain, because we treat fs.NotExist as a non-error in other places (.stignore may not exist).

Reply:
Ah got it, thanks for the explanation. Maybe instead of the comment make it explicit by checking:

```suggestion
		if fs.IsNotExist(err) {
			err = errors.New("missing included file")
		}
		return nil, parseError(err)
```

I am fine either way, just a suggestion.